packages/compass-e2e-tests/tests/data-modeling-tab.test.ts

```diff
@@ -97,13 +121,33 @@ async function getDiagramNodes(
   return nodes;
 }
 
+async function getDiagramEdges(
+  browser: CompassBrowser,
+  expectedCount: number
+): Promise<Edge[]> {
+  let edges: Edge[] = [];
+  await browser.waitUntil(async () => {
+    edges = await browser.execute(function (selector) {
+      const node = document.querySelector(selector);
+      if (!node) {
+        throw new Error(`Element with selector ${selector} not found`);
+      }
+      return (
+        node as Element & { _diagram: DiagramInstance }
+      )._diagram.getEdges();
+    }, Selectors.DataModelEditor);
+    return edges.length === expectedCount;
+  });
+  return edges;
+}
+
 /**
  * Moves a node to the specified coordinates and returns its original position.
  */
 async function moveNode(
   browser: CompassBrowser,
   selector: string,
-  coordinates: { x: number; y: number }
+  coordinates: { x: number; y: number; origin?: 'pointer' | 'viewport' }
```

Comment:
[nit]
```suggestion
  pointerMoveAction: { x: number; y: number; origin?: 'pointer' | 'viewport' }
``` 
or something as its not just _coordinates_ now

Reply:
Makes sense, thanks!